master/manifests/site.pp

```diff
@@ -439,7 +439,7 @@
 
 # Add jenkins user to docker group if not already
 exec {"jenkins docker membership":
-  unless => "/bin/grep -q 'docker\\S*jenkins' /etc/group",
+  unless => "/bin/bash -c '/usr/bin/id -nG jenkins | /bin/grep -wq docker'",
```

Comment:
It's using the pipe operator to get the content to grep, I didn't find a way to get it into one command without bashisms.